source/howto/mobility/getstarted.rst

```diff
@@ -445,3 +448,26 @@ of values, respectively:
 * :literal:`where=smetadata.capacity.gt.100,smetadata.municipality.eq."Bolzano -
   Bozen"` same as previous query, but only parking lots in Bolzano are shown.
 
+Additional Parameters
+~~~~~~~~~~~~~~~~~~~~~
+
+There are a couple of other parameter that can be given to the API
+calls and are described in this section.
+
+.. rubric:: :literal:`shownull`
+         
+In order to show :strong:`null` values in the output of a query, add
+:literal:`shownull=true` ad the end of your query.
+
+.. rubric:: :literal:`distict`
+
+Results in query responses contain unique results, that is, if for
+some reason one element is retrieved multiple times while the query is
+executed, it will be nonetheless shown only once, for performance
+reasons. It is however possible to retrieve each single result and
```

Comment:
It is actually exactly the opposite. Showing duplicates costs less in terms of query performance, although the download might be bigger, but nowadays with fast internet connections, duplicate elimination is slower than downloading duplicates, except for some very large data retrievals...
However, for small data sets distinct is perfectly fine to be set to true... it is always a thought about download size vs query performance

Reply:
Which is the default behaviour when distinct is not explicited in the query?

Reply:
Default is distinct=true